support-frontend/assets/pages/digital-subscription-landing/digitalSubscriptionLanding.jsx

```diff
@@ -132,13 +132,16 @@ function LandingPage() {
       header={<CountrySwitcherHeader />}
       footer={footer}
     >
-      <DigitalHero
-        orderIsAGift={orderIsAGift}
-        countryGroupId={countryGroupId}
-        promotionCopy={sanitisedPromoCopy}
-        showPriceCards={showPriceCardsInHero}
-        priceList={heroPriceList}
-      />
+      {countryGroupId === AUDCountries || orderIsAGift ?
```

Comment:
There's a question about whether we should roll this out to the gift page too I guess?

Reply:
Yes, that's a good point - I'll check.

Reply:
After a long discussion, we concluded that we would keep the gift version of the page as it is (with an image rather than price cards).